### DISCUSSION TITLE
Unsupported 'react-native' version/range: ^0.69.0

### DISCUSSION THREAD

**Discussion by vicke4** [microsoft/rnx-kit#1679] [Q&A] [answered]
I can see that a profile to upgrade to 0.69 is added with #1447. But, when I try `npx @rnx-kit/dep-check --set-version 0.69`, I get `Unsupported 'react-native' version/range: ^0.69.0`. Is this because it is not published on NPM yet?

**Comment by tido64** [comment, marked as answer]
Sorry for the wait. We were waiting for more changes to go in before publishing. It should be released now: https://github.com/microsoft/rnx-kit/releases/tag/%40rnx-kit/dep-check%401.12.17

**Reply by vicke4** [reply]
Thanks!!
